src/motivation.md

```diff
@@ -0,0 +1,48 @@
+# Motivation
+
+The decentralized internet demands privacy-respecting data provenance!
+True data provenance will make data truly portable and allow users to share it with others as they see fit.
+
+### Non-repudiation: TLS is not enough
+
+![](png-diagrams/data_provenance_none.png)
+
+Transport Layer Security (TLS) plays a crucial role in digital security. TLS protects communication against eavesdropping and tampering. It ensures that the data received by the `User` from the `Server` is both **authentic** and **unchanged**. The data's authenticity is verified by the `User` through trusted Certificate Authorities (CAs). Data integrity is maintained by transmitting a cryptographic hash alongside the data, which safeguards against deliberate alterations.
```

Comment:
regarding: ... is both authentic and unchanged.

Doesn't the former imply the latter?